src/anjaliS/MeetingRoom.java

```diff
@@ -0,0 +1,32 @@
+package anjaliS;
+
+public class MeetingRoom {
+	String meetingRoomName;
+	int periodOfTime;
+
+	void bookMeetingRoom() {
+		meetingRoomName = "Nalanda";
+		periodOfTime = 1;
+
+	}
+
+	void bookMeetingRoom(String meetingRoomName, int periodOfTime) {
```

Comment:
This is not as per the expectation,

Write another method which helps to book Other then “Nalanda” meeting room by giving name of meeting room and hrs as parameter

you should restrict user to book nalanda right?
this is not simple as simple as you write, need to put some condition